tools/ci/templates/common.yml

```diff
@@ -0,0 +1,69 @@
+# [Template] Common build tasks shared between CI builds and PR validation.
+
+steps:
+# Build Native Plugins
+- template: nativesetup.yml
+  parameters:
+    ProjectPath: '${{ parameters.ProjectPath }}'
+
+# Build UWP ARM IL2CPP
+- template: unitybuild.yml
+  parameters:
+    ProjectPath: '${{ parameters.ProjectPath }}'
+    ToolsAssetsPath: '${{ parameters.ToolsAssetsPath }}'
+    Platform: 'UWP'
+    Arch: 'ARM'
+    ScriptingBackend: 'IL2CPP'
+    SceneList: '${{ parameters.ProjectPath }}\Assets\MixedReality-SpectatorView\SpectatorView\Scenes\SpectatorView.HoloLens.unity'
+    Define: ''
+    UnityArgs: ''
+
+# # Build UWP ARM IL2CPP ASA QR
```

Comment:
Any reason for leaving this commented out instead of removing?

Reply:
ah, i want to uncomment these additional passes once one succeeds on an agent. Good catch/I should probably enable these now.

FWIW - these builds are currently failing because our agents dont support symbolic linked directories. were working to fix this now.